core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -35,6 +51,14 @@ public Libc load() {
         return new EmptyLibc();
       }
     }
+
+    private boolean isGraalBuildTime() {
+      return System.getProperty(GRAAL_STATUS_PROP).equals(GRAAL_BUILDTIME_STATUS);
```

Comment:
This throws an NPE outside of Graal, `getProperty` returns null if the property does not exist.

Reply:
:facepalm: Fixed, good catch